ProjectLighthouse/Migrations/20220916141401_ScoreboardAdvSlot.cs

```diff
@@ -0,0 +1,23 @@
+using LBPUnion.ProjectLighthouse;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace ProjectLighthouse.Migrations
+{
+    [DbContext(typeof(Database))]
+    [Migration("20220916141401_ScoreboardAdvSlot.cs")]
```

Comment:
```suggestion
    [Migration("20220916141401_ScoreboardAdvSlot")]
```
I haven't realized this until now.

Reply:
I also did not realize this either